CSV/brands.csv

```diff
@@ -26,19 +26,19 @@ avc1,Advanced Video Coding extensions,ISO
 avci,AVC image and image collection brands,HEIF
 avcs,AVC image sequence brands,HEIF
 bbxm,Blinkbox Master File: H.264 video and 16-bit little-endian LPCM audio,Blinkbox
-CAEP,Canon Digital Camera,Canon
-CDes,Convergent Designs,Convergent
 caaa,CMAF Media Profile - AAC Adaptive Audio,CMAF
 caac,CMAF Media Profile - AAC Core,CMAF
+CAEP,Canon Digital Camera,Canon
```

Comment:
Not sure if the chosen order was not using upper case first and then lower case. Maybe should be in a different PR?

Reply:
Which command is used to sort?

Reply:
Also, something (Excel) thinks that DECE10 was a date and changed it to 10-Dec. Maybe re-do the PR without changing rows you don't intend to touch? (and we will work out the canonical sort separately).